<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>336</width>
    <height>248</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Shkarupa Mykyta</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="sortButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>140</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Open the sorting demonstration window&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Sort</string>
    </property>
   </widget>
   <widget class="QPushButton" name="listButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Single Linked List</string>
    </property>
   </widget>
   <widget class="QPushButton" name="splayTree">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>90</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Splay tree</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>336</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDemo">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionList"/>
    <addaction name="actionSplayTree"/>
    <addaction name="actionSort"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="action"/>
   </widget>
   <addaction name="menuDemo"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionContainer">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Container</string>
   </property>
  </action>
  <action name="actionSet">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Set</string>
   </property>
  </action>
  <action name="actionSort">
   <property name="text">
    <string>Sort</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionList">
   <property name="text">
    <string>Single Linked List</string>
   </property>
  </action>
  <action name="actionSplayTree">
   <property name="text">
    <string>Splay tree</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
